src/listener/ClusterViewListenerService.ts

```diff
@@ -73,7 +73,6 @@ export class ClusterViewListenerService {
         invocation.eventHandler = handler;
 
         this.logger.trace('ClusterViewListenerService', `Register attempt of cluster view handler to ${connection}`);
-        this.clusterService.clearMemberListVersion();
```

Comment:
I don't think the Java PR removes this.

Reply:
I checked on Java side and added `this.clusterService.onClusterConnect();` to fix.